src/hotspot/share/opto/library_call.cpp

```diff
@@ -5931,53 +5931,7 @@ bool LibraryCallKit::inline_multiplyToLen() {
     // 'x_start' points to x array + scaled xlen
     // 'y_start' points to y array + scaled ylen
 
-    // Allocate the result array
     Node* zlen = _gvn.transform(new AddINode(xlen, ylen));
```

Comment:
Would could generate one less instruction in the code cache if we did this `add` in the native runtime function.